packages/core/src/awsService/appBuilder/serverlessLand/metadataManager.ts

```diff
@@ -48,12 +41,17 @@ export class MetadataManager {
 
     public static initialize(): MetadataManager {
         const instance = MetadataManager.getInstance()
-        instance.loadMetadata(MetadataManager.metadataPath).catch((err) => {
+        const metadataPath = instance.getMetadataPath()
+        instance.loadMetadata(metadataPath).catch((err) => {
             throw new ToolkitError(`Failed to load metadata: ${err}`)
         })
         return instance
     }
 
+    public getMetadataPath(): string {
+        return globals.context.asAbsolutePath(path.join('dist', 'src', 'serverlessLand', 'metadata.json'))
```

Comment:
There is 100% never a reason to use join() on string literals.
```suggestion
        return globals.context.asAbsolutePath('dist/src/...'))
```